Repo.rb

```diff
@@ -44,26 +69,22 @@ def self.highest_above(dir)
       cur_dir = File.dirname(cur_dir)
     end
 
-    # res_dir = dir
-    # cur_dir = dir
-    # loop do
-    #   cur_dir = File.dirname(res_dir)
-    #   if is_repo_dir?(cur_dir)
-    #     res_dir = cur_dir
-    #     end
-    #   break if cur_dir == "/"
-    # end
     Repo.from_dir(res_dir)
   end
 
-  def self.from_repoInfo(repoInfo)
-    subs = repoInfo.subrepos
-    subRepos = subs.map(&:from_repoInfo)
-    Repo.new(repoInfo.path_to, repoInfo.subrepos, repoInfo.is_head, repoInfo.bindings)
+  def self.from_repo_info(repo_info)
+    subs = repo_info.subrepos
+    subRepos = subs.map(&:from_repo_info)
```

Comment:
Useless assignment to variable - `subRepos`.<br>Use snake_case for variable names.